ui/raidboss/data/04-sb/eureka/eureka_hydatos.ts

```diff
@@ -77,12 +77,18 @@ const triggerSet: TriggerSet<Data> = {
     },
   ],
   triggers: [
+    // https://xivapi.com/LogMessage/916
+    // en: 7 minutes have elapsed since your last activity. [...]
+    // But no 0x29 log line; so have to use GameLog for now.
     {
       id: 'BA Falling Asleep',
       type: 'GameLog',
       netRegex: { line: '7 minutes have elapsed since your last activity..*?', capture: false },
       response: Responses.wakeUp(),
     },
+    // https://xivapi.com/LogMessage/9069
+    // en: The memories of heroes past live on again!
+    // But no 0x29 log line; so have to use GameLog for now.
```

Comment:
This one's weird, there's no direct network packet referencing this so I think it's fired as part of some timeline effects or something.

Reply:
> This one's weird, there's no direct network packet referencing this so I think it's fired as part of some timeline effects or something.

That is weird.  IIRC, it would be emitted whenever Spirit of the Remembered procs and revives the player.  The proc chance would have to be server side I would think, so there's probably at least some network packet that signals the game client that the revive has proc'd, even if the actual game log line is being written solely by the client.  I'll see if I can pop into Eureka today/tomorrow and get some more info.

Reply:
Looked at this further, and there's nothing obvious that I can see that would signal a rez attributable to a Spirit of the Remembered proc.  The 0x0C `PlayerStats` line and gained effects like Weakness/Transcendence aren't specific to Remembrance.  So I think this is another one that's going to have to stay as-is.

Reply:
Well I finally found the network packet indicating this one, it's ActorControlSelf category 527 (0x20F):
```
4096|2024-03-18T12:34:52.2740000-04:00|ActorControlSelf|10001234|527|0000236D|00000000|00000669|00000000|00000000|00000000|00000000|450d8d6ada5c140f
00|2024-03-18T12:34:51.0000000-04:00|0939||The memories of heroes past live on again!|e710cf321339b40d
```

So another potential item to add to the ActorControl packet handler.

Reply:
> Well I finally found the network packet indicating this one, it's ActorControlSelf category 527 (0x20F):
> 
> ```
> 4096|2024-03-18T12:34:52.2740000-04:00|ActorControlSelf|10001234|527|0000236D|00000000|00000669|00000000|00000000|00000000|00000000|450d8d6ada5c140f
> 00|2024-03-18T12:34:51.0000000-04:00|0939||The memories of heroes past live on again!|e710cf321339b40d
> ```
> 
> So another potential item to add to the ActorControl packet handler.

Works for me.